src/ui/settings-tab.ts

```
@@ -200,5 +200,61 @@ export class CustomSlidesSettingTab extends PluginSettingTab {
             await this.plugin.saveSettings();
           }));
     });
+
+    generalGroup.addSetting((setting: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Use sentence case for UI text.

(obsidianmd/ui/sentence-case)